src/components/Menu.vue

```diff
@@ -0,0 +1,178 @@
+<template>
+  <popover :visible="visible" :align="alignLeft" bottom @closed="$emit('closed')">
+    <div class="menu__body">
+      <transition name="slide-left" mode="out-in">
+        <div v-if="visiblePanel == 1">
+          <div class="menu__panel">
+            <div
+              v-for="button in buttons"
+              :key="button.label"
+              class="menu__option"
+              @click="button.onclick"
+              v-html="button.html"
+            />
+            <div
+              v-if="otherButtons.length > 0"
+              class="menu__option menu__option--top-bordered"
+              @click="visiblePanel = 2"
+            >
+              Other Operations
```

Comment:
```suggestion
              Other operations
```
Sorry for being a typo-freak :p